packages/data-explorer-ui/src/components/Error/error.tsx

```diff
@@ -35,6 +67,21 @@ export const Error = ({ rootPath }: ErrorProps): JSX.Element => {
             </Link>
           </SectionActions>
         )}
+        {(requestUrlMessage || errorMessage) && (
+          <ErrorDetailsWrapper>
+            <Divider />
+            {requestUrlMessage && (
+              <ErrorDetailSection
+                title="Request URL"
+                detail={requestUrlMessage}
+              />
+            )}
+
+            {errorMessage && (
+              <ErrorDetailSection title="Error Message" detail={errorMessage} />
```

Comment:
This is nit - we love to organise our props and styles alphabetically, would you mind doing this here for the ErrorDetailSection props (and then also in the styles anywhere that needs re-arranging). Thank you!